src/gui/pointcloud/qgspointcloudrendererpropertieswidget.cpp

```diff
@@ -191,6 +224,17 @@ void QgsPointCloudRendererPropertiesWidget::syncToLayer( QgsMapLayer *layer )
     {
       mLabels->setChecked( mLayer->renderer()->showLabels() );
       mLabelOptions->setTextFormat( mLayer->renderer()->labelTextFormat() );
+      mZoomOutOptions->setCurrentIndex( static_cast<int>( mLayer->renderer()->zoomOutBehavior() ) );
```

Comment:
Can it be the case that we have a renderer with a zoomOutBehavior that was not added to the combobox? (when loading a project maybe?)

Reply:
To my knowledge no, for populating and also for setting the combo box we use the mLayer info.